pyproject.toml

```diff
@@ -37,6 +37,9 @@ dependencies = [
     "s3fs~=2024.9.0",
     "xarray~=2024.9.0",
     "geojson-pydantic>=1.1.1",
+    "python-dateutil>=2.9.0.post0",
+    "httpx>=0.27.2",
+    "pillow>=11.0.0",
```

Comment:
I've never tried but it seems that GDAL can create GIF natively https://gdal.org/en/latest/drivers/raster/gif.html 

I fear that adding pillow dependency will make the package size 💥 (we could also remove some dependencies like S3FS)

maybe we should ask @kylebarron to add python bindings to https://github.com/image-rs/image-gif 🙈

Reply:
> we could also remove some dependencies like S3FS

Yes!
 
> maybe we should ask @kylebarron to add python bindings to [image-rs/image-gif](https://github.com/image-rs/image-gif) 🙈

More rust!!

Reply:
I think GDAL can only write a single image to a GIF. I tried making a vrt out of two PNGs and then ran `gdal_translate` with `-of GIF` and got an error:
```bash
$ gdalbuildvrt -separate /tmp/test.vrt /tmp/2023-11-01T00\:00\:01Z.png /tmp/2024-05-15T00\:00\:01Z.png
0...10...20...30...40...50...60...70...80...90...100 - done.

$ gdal_translate -of GIF /tmp/test.vrt /tmp/test.gif
Input file size is 1440, 720
ERROR 6: GIF driver only supports one band images.
```

I ran `cdk synth` locally and compared the size of the zipped asset outputs in `cdk.out/` on the `develop` branch (without the `Pillow` dependency) and on this branch (with `Pillow`) and it looks like it only increases the size of the zipped code package by ~4MB so it might be ok.

@vincentsarago would it be ok if we keep `Pillow` in this PR and see how it looks when we deploy it? It would be great to keep it slim with `image-rs/image-gif` but if `Pillow` works for now that would be ideal.

Reply:
>  it looks like it only increases the size of the zipped code package by ~4MB so it might be ok.

🙏 looks ok then :-)